policy/beta/sbom/sbom_oci.rego

```diff
@@ -0,0 +1,15 @@
+package lib.sbom.oci
+
+import data.lib
+import data.lib.tkn
+import rego.v1
+
+_cyclonedx_sboms := [sbom |
+	some attestation in lib.pipelinerun_attestations
+	some task in tkn.build_tasks(attestation)
+
+	blob_ref := tkn.task_result(task, "SBOM_BLOB_URL")
+	blob := ec.oci.blob(blob_ref)
```

Comment:
I think this function has been around for long enough that we can probably bring it out of beta?

Reply:
I wasn't too sure when we added it, but it was last year in October, so yeah, I'll move it to the release...